modules/UI/UI.js

```diff
@@ -1267,7 +1269,11 @@ UI.showDeviceErrorDialog = function (micError, cameraError) {
 
     message = `${message}${doNotShowWarningAgainSection}`;
 
-    messageHandler.openDialog(
+    // To make sure we don't have multiple error dialogs open at the same time,
+    // we will just close the previous one if we are going to show a new one.
+    deviceErrorDialog && deviceErrorDialog.close();
+
+    deviceErrorDialog = messageHandler.openDialog(
```

Comment:
No. Do you think this can lead to some memory leaks? Well, I can add close handler to dialog to clear the reference. But still in the worst case we will have only one single instance of this dialog in memory.